typespec-tests/src/main/java/com/_specs_/azure/clientgenerator/core/access/implementation/models/AbstractModel.java

```diff
@@ -47,6 +75,7 @@ public String getName() {
     public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
```

Comment:
@alzimmermsft @haolingdong-msft @srnagar 

We need `@Generated` on this method.

Else, partial-update would be in a mess.